docs/configure-artifact-repository.md

```diff
@@ -18,11 +18,11 @@ Subsequent sections will show how to use it.
 
 ## Configuring Minio
 
-```
-$ brew install helm # mac, helm 3.x
-$ helm repo add stable https://kubernetes-charts.storage.googleapis.com/ # official Helm stable charts
-$ helm repo update
-$ helm install argo-artifacts stable/minio --set service.type=LoadBalancer --set fullnameOverride=argo-artifacts
+```bash
+brew install helm # mac, helm 3.x
```

Comment:
Hmm, we use the `$ ...` convention across our docs (even in other places in this same file). In my opinion, what we currently have is easier to parse and understand.

Do you have a good reason to remove the `$ ...` convention?